packages/icons/converter/index.js

```diff
@@ -7,8 +7,8 @@ const indexTemplate = require('./template.index.js')
 const fs = require('fs/promises')
 const { extname, resolve } = require('path')
 
-const SRC_DIR = resolve(__dirname, '../src/svg')
```

Comment:
looks like whole "converter" folder should move to top-level (as it should be out of build scope), but we will fix it later